client/layout/guided-tours/positioning.ts

```diff
@@ -80,23 +80,24 @@ const dialogPositioners = {
 	} ),
 };
 
-export const query = selector => [].slice.call( window.document.querySelectorAll( selector ) );
+export const query = ( selector: string ) =>
+	Array.from( window.document.querySelectorAll( selector ) );
 
-export const posToCss = ( { x, y } ) => ( {
+export const posToCss = ( { x, y }: Coordinate ): Pick< CSSProperties, 'top' | 'left' > => ( {
 	top: y ? y + 'px' : undefined,
 	left: x ? x + 'px' : undefined,
 } );
 
-function hasNonEmptyClientRect( el ) {
+function hasNonEmptyClientRect( el: Element ) {
 	const rect = el.getBoundingClientRect();
-	return ! ( rect.x === 0 && rect.y === 0 && rect.width === 0 && rect.height === 0 );
```

Comment:
This may have always been `true` in IE because `getBoundingClientRect` returns a `ClientRect` in IE rather than a DOMRect, the difference being additional `{x, y}` in `DOMRect`. Thanks TypeScript!